packages/plugin-ext/src/main/browser/menus/menus-contribution-handler.ts

```diff
@@ -44,6 +44,7 @@ import { TIMELINE_ITEM_CONTEXT_MENU } from '@theia/timeline/lib/browser/timeline
 import { TimelineItem } from '@theia/timeline/lib/common/timeline-model';
 import { COMMENT_CONTEXT, COMMENT_THREAD_CONTEXT, COMMENT_TITLE } from '../comments/comment-thread-widget';
 import { QuickCommandService } from '@theia/core/lib/browser';
+import { FileTreeWidget } from '@theia/filesystem/src/browser';
```

Comment:
Imports from `src` are discouraged, please import from `lib` instead. I think this is the source of the building error.